### PR TITLE
14.0 mig hr timesheet task stage

### PR DESCRIPTION
**Pull request by paidykumar-dreambits**
Migration from version 13.0 to 14.0

### CODE REVIEW

File: hr_timesheet_task_stage/__manifest__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 {
     "name": "Task Log: Open/Close Task",
-    "version": "13.0.1.1.0",
+    "version": "14.0.1.1.0",
```
Comment (dsolanki-initos) [inline hr_timesheet_task_stage/__manifest__.py:11]: Version must be as per oca guideline

File: hr_timesheet_task_stage/models/account_analytic_line.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            line.task_id.write({"stage_id": stage.id})
+
+    def action_close_task(self):
+        ProjectTaskType = self.env["project.task.type"]
+
+        for line in self.filtered("task_id.project_id"):
+            stage = ProjectTaskType.search(
+                [
+                    ("project_ids", "=", line.task_id.project_id.id),
+                    ("closed", "=", True),
```
Comment (dsolanki-initos) [inline hr_timesheet_task_stage/models/account_analytic_line.py:44]: Model "project.task.type"  don't have field "closed". Have you tested does it work?

File: hr_timesheet_task_stage/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "name": "Task Log: Open/Close Task",
+    "version": "14.0.1.1.0",
+    "category": "Operations/Timesheets",
+    "website": "https://github.com/OCA/timesheet",
+    "author": "Tecnativa, " "CorporateHub, " "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "application": False,
+    "summary": "Open/Close task from corresponding Task Log entry",
+    "depends": ["hr_timesheet", "project_stage_closed"],
```
Comment (dsolanki-initos) [inline hr_timesheet_task_stage/__manifest__.py:19]: In v14 we dont need the module "project_stage_closed" in dependency in base we already have is_closed field for "project.task.type" model

**Comment by dsolanki-initos** [comment]
@paidykumar-dreambits is it possible to fix the mentioned issues.?

**Comment by dsolanki-initos** [comment]
@paidykumar-dreambits can you please have a look at the suggesed issue. So that we can get the PR merge soon.

**Comment by dsolanki-initos** [comment]
Hi @paidykumar-dreambits It would be great if you can have a look at the above comments.

**Comment by dsolanki-initos** [comment]
Hello @paidykumar-dreambits  Can you please have a look at comments and resolve it.

**Comment by dsolanki-initos** [comment]
We didn't get any response since long I have done the required changes in new PR #446

**Comment by pedrobaeza** [comment]
Superseded by #446

**Pull request closed without merging**